test/prettier.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+/* eslint-disable node/shebang */
```

Comment:
It works actually, you just don't get pretty output if there's an error. But I'll remove it since that's an odd use case.